config.lua.dist

```diff
@@ -101,11 +101,13 @@ rateExp = 5
 rateSkill = 3
 rateLoot = 2
 rateMagic = 3
-rateSpawn = 1
 
 -- Monsters
+-- spawnIntervalRate controls the rate of monster spawning interval
+-- (spawntime divided by spawnIntervalRate) 1 = 100%, 2 = 50%, etc
 deSpawnRange = 2
 deSpawnRadius = 50
+spawnIntervalRate = 1
```

Comment:
If you make it 0, server will crash. Force value as 1 using std:max(1, configVal)